waffle-mock-contract/test/direct.test.ts

```diff
@@ -43,4 +44,21 @@ describe('Mock Contract - Integration (called directly)', () => {
     await expect(mockCounter.add(2)).to.be.revertedWith('Mock revert');
     expect(await mockCounter.add(3)).to.equal(1);
   });
+
+  it('should be able to call to another contract', async () => {
```

Comment:
There should also be a test for specifying just a function name, ignoring the arguments, so `read` instead of `read()`